examples/billing/get_invoices.rb

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# This code example retrieves the invoices issued last month for a given
+# billing setup.
+
+require 'optparse'
+require 'google/ads/google_ads'
+
+def get_invoices(customer_id, billing_setup_id)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  # Gets the date one month before now.
+  last_month = Date.today.prev_month
+
+  # Issues a request to list invoices.
+  response = client.service.invoice.list_invoices(
+    customer_id: customer_id,
+    billing_setup: client.path.billing_setup(customer_id, billing_setup_id),
+    # The year needs to be 2019 or later.
```

Comment:
Is this hard-coded as 2019 or is it a rolling window some distance in the past?

Reply:
It is hard-coded.